ema_workbench/em_framework/callbacks.py

```diff
@@ -138,7 +144,7 @@ def __init__(
 
         Parameters
         ----------
-        uncs : list
+        uncertainties : list
```

Comment:
I feel that `uncertainties` and `parameters` are mixed up here, because above `parameters = uncertainties + levers`

Reply:
sorry I don't follow. 

We pass uncertainties, levers and outcomes as separate arguments here, so what are you driving at?

Reply:
I think I read the code wrong.

The docs now only state _uncertainties_ and _outcomes_ as input parameters, should _levers_ and _nr_experiments_ be added?

Reply:
good point. I have fixed the docs in the latest commit